prow/cmd/hook/main.go

```diff
@@ -51,8 +51,8 @@ var (
 	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
 	pluginConfig = flag.String("plugin-config", "/etc/plugins/plugins", "Path to plugin config file.")
 
-	local = flag.Bool("local", false, "Run locally for testing purposes only. Does not require secret files.")
-	dry   = flag.Bool("dry", false, "Dry run for testing. Uses API tokens but does not mutate.")
+	local  = flag.Bool("local", false, "Run locally for testing purposes only. Does not require secret files.")
+	dryRun = flag.Bool("dry-run", false, "Dry run for testing. Uses API tokens but does not mutate.")
```

Comment:
At least for plank, the default is true. If you set it to true, be sure to manually set it to false in the deployment.